src/libraries/System.Private.CoreLib/src/System/Reflection/ImageFileMachine.cs

```diff
@@ -9,5 +9,6 @@ public enum ImageFileMachine
         IA64 = 0x0200,
         AMD64 = 0x8664,
         ARM = 0x01c4,
+        LOONGARCH64 = 0xdd64,
```

Comment:
This enum is deprecated. We are not adding new architectures to this enum. This change should be reverted. More details https://github.com/dotnet/runtime/issues/58970#issuecomment-917417493

Reply:
OK.Thanks
I will update it later.

It's best that .NET defines a specification about the  enum for different CPUs.